src/Searching.js

```diff
@@ -48,37 +88,44 @@ async function combinedSearch(searchTerm) {
     const localResult = await localPromise;
     const serverSideResult = await serverSidePromise;
 
-    // Combine the search results into one result.
-    const result = {};
+    const serverQuery = serverSideResult.query;
+    const serverResponse = serverSideResult.response;
 
-    // Our localResult and serverSideResult are both ordered by
-    // recency separately, when we combine them the order might not
-    // be the right one so we need to sort them.
-    const compare = (a, b) => {
-        const aEvent = a.context.getEvent().event;
-        const bEvent = b.context.getEvent().event;
+    const localQuery = localResult.query;
+    const localResponse = localResult.response;
 
-        if (aEvent.origin_server_ts >
-            bEvent.origin_server_ts) return -1;
-        if (aEvent.origin_server_ts <
-            bEvent.origin_server_ts) return 1;
-        return 0;
+    // Store our queries for later on so we can support pagination.
+    const emptyResult = {
+        seshatQuery: localQuery,
+        _query: serverQuery,
```

Comment:
Maybe `serverQuery` as a key as well, instead of `_query` especially when there are two queries?